client/src/components/ScoreDisplay/ScoreDisplay.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../scss/index.scss";
+
+.tableScore {
+  color: $black-color;
+  width: 100%;
+
+  thead {
+    display: none;
```

Comment:
is removed on DOM to accessibility assistance. Privilegiate visibility hidden or another method to mask this